data/data/bootstrap/files/usr/local/bin/installer-gather.sh

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+ARTIFACTS="${1:-/tmp/artifacts}"
+
+echo "Gathering bootstrap journals ..."
+mkdir -p "${ARTIFACTS}/bootstrap/journals"
+for service in bootkube openshift kubelet crio
+do
+    journalctl --boot --no-pager --output=short --unit="${service}" > "${ARTIFACTS}/bootstrap/journals/${service}.log"
+done
+
+echo "Gathering bootstrap containers ..."
+mkdir -p "${ARTIFACTS}/bootstrap/containers"
+sudo crictl ps --all --quiet | while read -r container
+do
+    container_name="$(sudo crictl ps -a --id "${container}" -v | grep -oP "Name: \\K(.*)")"
```

Comment:
Should we stay consistent with options? `-a` is the same as `--all`
Do we need to run this command twice? Why don't we get the json output, or parse this information line by line? 

```
sudo crictl ps --all | grep -v CONTAINER_ID | while read -r container_line
do
    container_name="$(awk '{printt $5}'' < container_line"      ### something like this
```
- This would save on having to make 2 crictl commands, for the information you fundamentally already have.